BFH_USZ_PICC/BFH_USZ_PICC/BFH_USZ_PICC.csproj

```diff
@@ -97,6 +92,11 @@
       <DesignTime>True</DesignTime>
       <DependentUpon>AppResources.it.resx</DependentUpon>
     </Compile>
+    <Compile Include="Resx\AppResources1.Designer.cs">
```

Comment:
With the "1" at the end, the system does not generate a new file if there are changes in the AppResources.rex file. 

So I propose to Exchange "AppRessources" with "AppRessources1", otherwise every time we will change something in AppResources.rex, an "AppRessources1" will be generated and we have to delete it manually.